src/compiler/crystal/codegen/exception.cr

```diff
@@ -4,197 +4,278 @@ class Crystal::CodeGenVisitor
   @node_ensure_exception_handlers = {} of UInt64 => Handler
 
   def visit(node : ExceptionHandler)
-    rescue_block = new_block "rescue"
-
-    node_rescues = node.rescues
-    node_ensure = node.ensure
-    node_else = node.else
-    rescue_ensure_block = nil
-
-    # Here we transform this:
+    # In this codegen, we assume that LLVM only provides us with a basic try/catch abstraction with no
+    # type restrictions on the exception caught. The basic strategy is to codegen this
     #
-    # ```
+    # ```cr
```

Comment:
`cr` here is invalid (that's gfm), it better be empty (like it was before)

Reply:
It doesn't hurt, though.

Reply:
It doesn't help either.

Reply:
who cares

Reply:
Yeah, and humans know what `cr` means...

Reply:
I agree with @Sija. It's better without the `cr`. But we can clean up that later (together with one more ```crystal that is [somewhere in the code](https://github.com/crystal-lang/crystal/blob/master/src/time.cr#L23)...)